platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -178,8 +178,9 @@ In style JSON | In the SDK
 `geojson`     | `MGLShapeSource`
 `raster`      | `MGLRasterSource`
 `vector`      | `MGLVectorSource`
+`image`       | `MGLImageSource`
```

Comment:
Note that this is an .ejs file, which means it’s a template for the runtime styling codegen tool. After running `make darwin-style-code`, you should see these changes propagate to the iOS- and macOS-specific Markdown files.